packages/FormElement/README.md

```diff
@@ -66,10 +96,17 @@ import MyInput from "./MyInput";
 - `label` (required)
 - `size`
 
+#### FormElement.Content
+
+- `children` (required)
```

Comment:
I'm not sure how to do it, but I think it would be helpful to document the arguments to the `children` function (`idForLabel`, `refLabel`, `ariaDescribedBy`).

Might also be useful to have that here:
https://github.com/acl-services/paprika/blob/ux-698-formElement-limit-supported-components/packages/FormElement/src/components/Content/Content.js#L7

...but for now I think the README is good enough.